### PR TITLE
Add Jinja-based `hostnames` to Zabbix inventory plugin

### PR DESCRIPTION
**Pull request by dmnagornyi**
Adds a `hostnames` option to `zabbix_inventory` to build `inventory_hostname` from Jinja expressions. The plugin now evaluates expressions via `Constructable._compose`, supports optional `{{ }}` wrapping, and raises a clear error on duplicate hostnames. Docs updated to describe the option and collision behavior.

### CODE REVIEW

**Comment by dmnagornyi** [comment]
Fixes https://github.com/zabbix/ansible-collection/issues/29

**Comment by dmnagornyi** [comment]
Hi! Just a friendly ping @AleksandrKotsegubov on this PR.

Whenever you have time, could you please take a look? If there’s anything you’d like me to adjust (tests/docs/style/edge cases) to get it merge-ready, I’m happy to update it.

**Comment by dmnagornyi** [comment]
Just checking on the status of this PR. If this repository is still the right place for enhancements to the zabbix_inventory plugin, I’m happy to iterate on any feedback.
If maintenance has moved elsewhere, please point me to the right process and I’ll rework it. Thanks!

File: plugins/inventory/zabbix_inventory.py [hunk 1]
```diff
@@ -58,6 +58,14 @@
         type: str
         default: 'zabbix_'
         description: Prefix to use for parameters given from Zabbix API.
+    hostnames:
+        type: list
+        elements: str
```
Comment (Andersson007) [inline plugins/inventory/zabbix_inventory.py:63]:
```suggestion
        elements: str
        version_added: '1.5.0'
```

**Review by Andersson007 — COMMENTED** [review]
@dmnagornyi hi, would be imo good to add some examples to the EXAMPLES block and a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). Thanks

**Comment by Andersson007** [comment]
> Just checking on the status of this PR. If this repository is still the right place for enhancements to the zabbix_inventory plugin, I’m happy to iterate on any feedback. If maintenance has moved elsewhere, please point me to the right process and I’ll rework it. Thanks!

To your question - no idea, i'm not a maintainer of this collection

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@dmnagornyi maybe it makes sense to mention people explicitly who merged others' PRs in this repo. FYI

**Comment by dmnagornyi** [comment]
Hey, @atocko do you happen to know if there are any active maintainers left for this repository?

**Comment by mgeneralova** [comment]
Hi @dmnagornyi, thank you for your contribution! The team is currently focused on other projects, but we plan to include this in the next Zabbix Ansible Collection release.

**Comment by dmnagornyi** [comment]
> Hi @dmnagornyi, thank you for your contribution! The team is currently focused on other projects, but we plan to include this in the next Zabbix Ansible Collection release.

Is the planned release date known? Or does it depend on reaching certain milestones?

**Comment by zjleblanc** [comment]
> > Hi @dmnagornyi, thank you for your contribution! The team is currently focused on other projects, but we plan to include this in the next Zabbix Ansible Collection release.
> 
> Is the planned release date known? Or does it depend on reaching certain milestones?

@mgeneralova we have Zabbix users very interested in this feature, I would also be interested in a timeline (even an estimate) for the next release.
